tutorials/06-physics-simulation-concepts.md

```diff
@@ -0,0 +1,240 @@
+    contacts or physically unfeasible contact forces.
+
+Ignition adopts SDFormat structure to describe not only the visual element but
+also the dynamic physics aspects. To get more in-depth of what you can define
+the environment in the SDFormat file, please refer to this
+[SDFormat specification](http://sdformat.org/spec?ver=1.7&elem=sdf).
+For a comprehensive tutorial for constructing your robot model as SDFormat,
+please refer to [this tutorial](https://ignitionrobotics.org/docs/citadel/building_robot).
+
+
+## Physics concepts in Ignition Gazebo simulation
+
+First, please see [Understand the GUI tutorial](https://ignitionrobotics.org/docs/citadel/gui)
+for an overview of Ignition Gazebo GUI. In this tutorial, we will show how to
+manipulate and visualize some physics aspects using interesting
+models powered by Ignition Physics on Ignition Gazebo.
+
+### Differential Drive
+
+This demo world demonstrates how we can control simulated differential drive on
+cars depending on physics engines and visualize the collision concept. Please
+download the Differential Drive simulation world demo to your home folder by:
+
+```bash
+wget https://raw.githubusercontent.com/ignitionrobotics/ign-gazebo/main/examples/worlds/diff_drive.sdf -P ~
+```
+
+We can now start our differential drive simulation on Ignition Gazebo by:
+
+```bash
+ign gazebo ~/diff_drive.sdf
+```
+
+To see the pose changing when the simulation running, we publish a
+\ref ignition::msgs::Twist "Twist" message to command the `vehicle_blue` car to
+move in a circle with 2 meters radius as follow:
+
+```bash
+ign topic -t "/model/vehicle_blue/cmd_vel" -m ignition.msgs.Twist -p "linear: {x: 1.0}, angular: {z: 0.5}"
+```
+
+Then please press the Play button to start the simulation.
+This command tells the car to move in its coordinate frame with velocity
+1.0 meter per second in the X-axis and angular velocity of 0.5 radians per
+second in Z-axis.
+
+Note that the mechanism to move the car is different depending on the used physics
+engine. Using [dartsim](https://github.com/ignitionrobotics/ign-physics/tree/master/dartsim),
+the car is moved by applying force on the joints, please see [DiffDrive.cc](https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo4/src/systems/diff_drive/DiffDrive.cc#L333) code.
+Using [TPE](https://github.com/ignitionrobotics/ign-physics/tree/main/tpe),
+TPE directly sets model velocity in [VelocityControl.cc](https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo4/src/systems/velocity_control/VelocityControl.cc#L117) code.
+
+#### Monitoring the model and its links pose
+
+We can monitor the model pose by selecting the moving car and then
+select the drop-down list `Pose`. Moreover, we could also read the model
+links' poses relative to their parent link by selecting the
+corresponding link on the model tree:
+
+<img src="https://user-images.githubusercontent.com/18066876/97189395-d45ef380-17a4-11eb-9c29-cf33b5b184c4.gif"/>
```

Comment:
Another tip for the future: the tutorials can refer to local images using `@image` like this:

https://github.com/ignitionrobotics/ign-rendering/blob/ign-rendering4/tutorials/18_simple_demo_tutorial.md

We just need to update `CMakeLists` to copy the images from tutorials:

https://github.com/ignitionrobotics/ign-rendering/blob/2c85d92f34f29e997aaf6b085ea376412032e5eb/CMakeLists.txt#L160